coralproject/talk
# [CORL-487] Update Spanish translations for v5

## What does this PR do?
- Adds Spanish translations 

## How do I test this PR?

- Change default language of Coral to "es"
- See copy display in Spanish
